com.github.micahflee.torbrowser-launcher.yaml

```diff
@@ -0,0 +1,123 @@
+app-id: com.github.micahflee.torbrowser-launcher
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+separate-locales: false
+command: torbrowser-launcher
+rename-desktop-file: torbrowser.desktop
+rename-appdata-file: torbrowser.appdata.xml
+rename-icon: torbrowser
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --persist=.config/torbrowser
+  - --persist=.local/share/torbrowser
```

Comment:
Is it hardcoded in the launcher or in TorBrowser itself? Is the former, maybe it's better to patch this, too?

Reply:
Should be in the launcher. However, I currently don't have enough time to look into the source code. Also, I didn't want to make any unnecessary modifications.

Reply:
I'd rather consider such changes at least highly desirable. Will look into it later.

Reply:
I opened a pr (https://github.com/AsciiWolf/flathub/pull/1) against this branch that should fix this

Reply:
Upstream PR: https://github.com/micahflee/torbrowser-launcher/pull/422